### PR TITLE
Update Translation `librart/typing.po` rst:457 ~ rst:762

### PR DESCRIPTION
**Pull request by rockleona**
<details>
<summary>別點會暴雷</summary>

image

image

</details>

### CODE REVIEW

**Comment by mattwang44** [comment]
@weijay0804 最近有空來幫忙 review 嗎？

**Comment by weijay0804** [comment]
> @weijay0804 最近有空來幫忙 review 嗎？

Ok 沒問題👌

File: library/typing.po [hunk 1]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
-msgstr ""
+msgstr "請記得 ``type[C]`` 是共變 (covariant) 的： ::"
 
 #: ../../library/typing.rst:485
 msgid ""
 "The only legal parameters for :class:`type` are classes, :data:`Any`, :ref:"
 "`type variables <generics>`, and unions of any of these types. For example::"
 msgstr ""
+":class:`type` 僅有的合法參數是類別、:data:`Any`、\\ :ref:`型別變數 "
+"<generics>`\\ 以及這些型別任意組合成的交集。舉例來說： ::"
```
Comment (weijay0804) [inline library/typing.po]:
```suggestion
"<generics>`\\ 以及這些型別任意組合成的聯集。舉例來說： ::"
```

這邊的 `unions` 我想應該是指集合中的 `聯集`。

截圖 2024-01-07 下午4 36 51

File: library/typing.po [hunk 2]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
 
 #: ../../library/typing.rst:497
 msgid ""
 "``type[Any]`` is equivalent to :class:`type`, which is the root of Python's :"
 "ref:`metaclass hierarchy <metaclasses>`."
 msgstr ""
+"``type[Any]`` 以及 :class:`type` 是相等的，其為 Python :ref:`metaclass 階層結"
+"構 (hierachy) <metaclasses>`。"
```
Comment (weijay0804) [inline library/typing.po]:
```suggestion
"``type[Any]`` 等價於 :class:`type` ，其為 Python :ref:`metaclass 階層結"
"構 (hierachy) <metaclasses>`。"
```

這樣寫會不會比較簡潔一點？

File: library/typing.po [hunk 3]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
+"泛型類別隱性繼承了 :class:`Generic`。為了相容 Python 3.11 及更早版本，也可以"
+"明確的繼承 :class:`Generic` 並指出是一個泛型類別： ::"
 
 #: ../../library/typing.rst:543
 msgid ""
 "Generic classes have :meth:`~object.__class_getitem__` methods, meaning they "
 "can be parameterised at runtime (e.g. ``LoggedVar[int]`` below)::"
 msgstr ""
+"泛型類別有 :meth:`~object.__class_getitem__` 方法，其意味著可以在 runtime 進"
+"行參數化（如下述的 ``LoggedVar[int]``）： ::"
```
Comment (weijay0804) [inline library/typing.po:690]:
```suggestion
"泛型類別有 :meth:`~object.__class_getitem__` 方法，其意味著可以在執行時期 (runtime) 進"
"行參數化（如下述的 ``LoggedVar[int]``）： ::"
```

這邊的 `runtime` 感覺可以翻譯出來，可以參考 [wiki 的執行時期頁面](https://zh.wikipedia.org/zh-tw/執行期)

File: library/typing.po [hunk 4]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
+msgstr "在這種情況下 ``MyDict`` 有一個單一的參數 ``T``。"
 
 #: ../../library/typing.rst:597
 msgid ""
 "Using a generic class without specifying type parameters assumes :data:`Any` "
 "for each position. In the following example, ``MyIterable`` is not generic "
 "but implicitly inherits from ``Iterable[Any]``:"
 msgstr ""
+"若使用泛型類別卻沒有特指型別參數，則會將每個位子視為 :data:`Any`。在下列的範"
+"例中 ``MyIterable`` 不是泛型，但隱性繼承了 ``Iterable[Any]``： ::"
```
Comment (weijay0804) [inline library/typing.po]:
```suggestion
"若使用泛型類別卻沒有特指型別參數，則會將每個位置視為 :data:`Any`。在下列的範"
"例中 ``MyIterable`` 不是泛型，但隱性繼承了 ``Iterable[Any]``： ::"
```

這邊翻譯成 `位置` 感覺比較順

File: library/typing.po [hunk 5]
```diff
@@ -801,18 +841,23 @@ msgid ""
 "declared to be of type :class:`str` and receives an :class:`int` value at "
 "runtime!"
 msgstr ""
+"請注意，當賦予型別為 :data:`Any` 的值更精確的型別時，將不會執行任何型別檢查。"
+"舉例來說，靜態型別檢查器不會在 runtime 中，將 ``a`` 賦值給 ``s`` 的情況下回報"
```
Comment (weijay0804) [inline library/typing.po:844]:
```suggestion
"舉例來說，靜態型別檢查器不會在執行時期中，將 ``a`` 賦值給 ``s`` 的情況下回報"
```

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195494700。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195494700.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195495964。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195495964.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195497005。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195497005.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195498090。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195498090.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195499151。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195499151.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195500436。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195500436.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195501471。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195501471.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195502805。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195502805.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

**Comment by llkj1** [comment]
感谢您与我们联系。(Thank you for contacting us.)
------------------------------------------------------------

我们收到了您的请求，并会在一至两个工作日内给您回复。您的案例编号是 102195504048。

想了解更多与开发相关主题的信息，请访问 开发者支持网页 (https://developer.apple.com/support/)。
                  
Apple Developer Program Support


We’ve received your support request and will get back to you in one to two business days. Your case number is 102195504048.

For additional information on development-related topics, visit:
https://developer.apple.com/support/

Best regards, 

Apple Developer Program Support
           

------------------------------------------------------------
Copyright (c) 2024 Apple Inc. All rights reserved.

Contact Us
https://developer.apple.com/contact/

Developer
https://developer.apple.com/

My Apple ID
https://appleid.apple.com

Privacy Policy
https://www.apple.com/privacy/

File: library/typing.po (hunk 5 above)
Comment (rockleona) [reply library/typing.po:844]: 同上面的留言

File: library/typing.po (hunk 3 above)
Comment (rockleona) [reply library/typing.po:690]: 我可能不會修改這邊喔，因為之前 Matt 幫我 Review  [這個 Issue](https://github.com/python/python-docs-zh-tw/pull/691#discussion_r1378450489) 的時候有提到

File: library/typing.po [hunk 6]
```diff
@@ -632,86 +632,103 @@ msgid ""
 "<Type>`) may accept values that are classes themselves -- specifically, it "
 "will accept the *class object* of ``C``. For example::"
 msgstr ""
+"一個變數被註解為 ``C`` 可以接受一個型別為 ``C`` 的值。相對的，一個變數備註解"
```
Comment (mattwang44) [inline library/typing.po]: 一個我自己很久以前未與他人討論過的翻譯習慣是：
- annotate -> 註釋
- comment -> 註解

也有看過其他人把 annotate 翻成「標注」，有什麼看法嗎？

File: library/typing.po [hunk 7]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
 msgid "Note that ``type[C]`` is covariant::"
-msgstr ""
+msgstr "請記得 ``type[C]`` 是共變 (covariant) 的： ::"
 
 #: ../../library/typing.rst:485
 msgid ""
 "The only legal parameters for :class:`type` are classes, :data:`Any`, :ref:"
 "`type variables <generics>`, and unions of any of these types. For example::"
 msgstr ""
+":class:`type` 僅有的合法參數是類別、:data:`Any`、\\ :ref:`型別變數 "
```
Comment (mattwang44) [inline library/typing.po]: nit 中文標點符號旁可以不用加斜線
```suggestion
":class:`type` 僅有的合法參數是類別、:data:`Any`、:ref:`型別變數 "
```

File: library/typing.po [hunk 8]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
 "``type[Any]`` is equivalent to :class:`type`, which is the root of Python's :"
 "ref:`metaclass hierarchy <metaclasses>`."
 msgstr ""
+"``type[Any]`` 等價於 :class:`type` ，其為 Python :ref:`metaclass 階層結"
+"構 (hierachy) <metaclasses>`。"
 
 #: ../../library/typing.rst:503
 msgid "User-defined generic types"
-msgstr ""
+msgstr "使用者定義泛用型別"
```
Comment (mattwang44) [inline library/typing.po]: 我之前翻譯成「泛型型別」，但一樣是沒和其他人討論過的決定，有什麼看法嗎？

File: library/typing.po [hunk 9]
```diff
@@ -632,86 +632,103 @@ msgid ""
...
+"泛型類別有 :meth:`~object.__class_getitem__` 方法，其意味著可以在 runtime 進"
+"行參數化（如下述的 ``LoggedVar[int]``）： ::"
 
 #: ../../library/typing.rst:552
 msgid ""
 "A generic type can have any number of type variables. All varieties of :"
 "class:`TypeVar` are permissible as parameters for a generic type::"
 msgstr ""
+"一個泛型型別可以有任意數量的型別變數。所有種類的 :class:`TypeVar` 都可以作為"
+"泛型行別的參數： ::"
```
Comment (mattwang44) [inline library/typing.po]: typo?

File: library/typing.po [hunk 10]
```diff
@@ -739,12 +758,18 @@ msgid ""
...
+"格式的參數規格變數來進行表示。對於上述作為參數規格變數的型別變數，將持續被型"
+"別模組視為一個特定的型別變數。對此，其中一個例外是一個型別列表可以替代 :"
+"class:`ParamSpec`： ::"
 
 #: ../../library/typing.rst:651
 msgid ""
 "Classes generic over a :class:`ParamSpec` can also be created using explicit "
 "inheritance from :class:`Generic`. In this case, ``**`` is not used::"
 msgstr ""
+"具有 :class:`ParamSpec` 的泛型類別可以透過顯示繼承 :class:`Generic` 進行建"
```
Comment (mattwang44) [inline library/typing.po]:
```suggestion
"具有 :class:`ParamSpec` 的泛型類別可以透過顯性繼承 :class:`Generic` 進行建"
```

File: library/typing.po (hunk 6 above)
Comment (rockleona) [reply library/typing.po]: > 一個我自己很久以前未與他人討論過的翻譯習慣是：
> 
> * annotate -> 註釋
> * comment -> 註解

我同意這邊的用法，
而且這邊是我的失誤，上次 PR 的時候我們有討論過用法了

所以往後都會維持這樣的翻譯

File: library/typing.po (hunk 8 above)
Comment (rockleona) [reply library/typing.po]: 我發現其他檔案跟 [術語表](https://docs.python.org/zh-tw/3/glossary.html#term-generic-type) 都是使用「泛型型別」

會進行修正

File: library/typing.po (hunk 3 above)
Comment (weijay0804) [reply library/typing.po:690]: 了解，沒問題～

File: library/typing.po (hunk 6 above)
Comment (weijay0804) [reply library/typing.po]: 喔喔，這裡用「註釋」的確比較貼近語意

File: library/typing.po (hunk 9 above)
Comment (weijay0804) [reply library/typing.po]: 拍謝，眼幹沒檢查到😅

**Review by weijay0804 — APPROVED** [review]
LGTM

感謝 Matt 大大～

**Review by mattwang44 — APPROVED** [review]

**Pull request merged**